live-examples/html-examples/content-sectioning/footer.html

```diff
@@ -0,0 +1,15 @@
+<h1>How to be a wizard</h1>
+<ol>
+    <li>Grow a long, majestic beard.</li>
+    <li>Wear a tall pointed hat. </li>
+    <li>Befriend the eagles.</li>
+    <li>Have I mentioned the beard?</li>
+</ol>
+<footer>
+    <p>&copy; 2018 Gandalf</p>
+    <ul>
+        <li><a href="http://example.com/terms">Terms</a></li>
+        <li><a href="http://example.com/blog">Blog</a></li>
+        <li><a href="http://example.com/about">About</a></li>
```

Comment:
@2alin Yes, let's make these just list items. Then you can also remove lines 32-40 of the CSS file. Thanks so much.